README.md

```diff
@@ -17,14 +17,16 @@ solution. You can find the sample code for the Android client code in the
 - Sign up for [Firebase](https://firebase.google.com/) and create a new project
   in the [Firebase console](https://console.firebase.google.com/).
 - Install the following tools in your development environment:
-  - [Python 2.7](https://www.python.org/downloads/)
   - [Java 8](https://java.com/en/download/)
   - [Apache Maven](https://maven.apache.org/)
   - [Google Cloud SDK](https://cloud.google.com/sdk/)
 
 > **Note**: Firebase is a Google product, independent from Google Cloud
 > Platform.
 
+> App Engine Flexible Environment [needs Java 8 Runtime](https://cloud.google.com/appengine/docs/flexible/java/setting-up-environment). However, in your local development environment you can 
```

Comment:
A few comments with this paragraph:

* It doesn't need to be formatted as a [quote][0], just a regular paragraph should work. This is especially important if there's another quote nearby, like in the previous paragraph.
* This reads as if *App Engine Flexible Environment needs the Java 8 runtime*, which it doesn't necessarily. App Engine Flexible Environment can use many runtimes, including Java 8.

The sample *needs* App Engine Flexible Environment with the Java 8 runtime. There might be a better way to phrase this. Could you help me understand what is your goal with this addition?

[0]: https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#quoting-text

Reply:
Updated the sentence based on your feedback. Thanks!